ores/wsgi/application.py

```diff
@@ -1,13 +1,17 @@
 import json
 import os
 
-from flask import Blueprint, Flask, request
-from revscoring.scorers import Scorer
 
-from . import routes
+def configure(config):
+    if 'data_path' in config and 'nltk' in config['data_path']:
+        import nltk
+        nltk.data.path.append(config['data_path']['nltk'])
 
+    from flask import Blueprint, Flask, request
+    from revscoring.scorers import Scorer
+
+    from . import routes
```

Comment:
Because they otherwise import nltk and try to use it without having set the path and it fails. And I need the config object to set the path.